### PR TITLE
Fix: (UI) Maintain search type and parameters on repeat search

### LINKED ISSUE #1317: When reusing search it does not include the paramters
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When you try to run a search with the same exact query as another query in the history using the specific button for it, it doesn't actually include the parameters.

image

Here you can see I relaunched the search of Fate/Stay by pressing the little button, but both of my latest search didn't include the "Season 1" parameter.

### Expected Behavior

When using this button, the query should be exactly the same.

### Steps To Reproduce

- Create a search query with parameters (such as Season)
- Try to run the search manually by pressing the little button in the search history.
- It will not use the parameters

### Environment

```markdown
- OS: Windows 10 Pro
- Prowlarr: 1.0.1.2220 
- Docker Install: No
- Using Reverse Proxy: No
- Browser:Firefuck 102+
```

### What branch are you running?

Master

### Trace Logs?

----------------------------------------------

### PR DESCRIPTION
**Pull request by mynameisbogdan**
#### Database Migration
YES - 034

#### Todos
- [x] Tests

#### Issues Fixed or Closed by this PR

* Fixes #1317

### CODE REVIEW

**Review by bakerboy448 — APPROVED** [review]
Seems fine here

File: frontend/src/History/HistoryRow.js [hunk 1]
```diff
@@ -44,15 +44,56 @@ class HistoryRow extends Component {
...
+      'tvMazeId',
+      'season',
+      'episode',
+      'artist',
+      'album',
+      'label',
+      'track',
+      'year',
+      'genre',
+      'author',
```
Comment (Qstick) [inline frontend/src/History/HistoryRow.js:71]: Can we do it without hard coding these? We already have to remember to update these in a few places on the backend that often get missed

Comment (bakerboy448) [reply frontend/src/History/HistoryRow.js:71]: backend and front end - anyway to have these in a singular or twoish spots?

Comment (Qstick) [reply frontend/src/History/HistoryRow.js:71]: Sure, I'll look at it when I have time.

Baker, lease avoid useless comments

Comment (Qstick) [reply frontend/src/History/HistoryRow.js]: Haha, now we've gone to 3 extra places in UI to update them 😭

Comment (Qstick) [reply frontend/src/History/HistoryRow.js]: Might be fine, I haven't looked close... Just saw more lists of parameters and head started spinning thinking of all the locations to add a new parameter

**Pull request merged**